Cargo.toml

```diff
@@ -116,30 +116,33 @@ rlp = { version = "0.5.1", default-features = false }
 substrate-bn = { version = "0.6.0", default-features = false }
 
 # IPLD/Encoding
-cid = { version = "0.10.1", default-features = false, features = [
-     "serde-codec",
+cid = { version = "0.11.1", default-features = false, features = [
+     "serde",
      "std",
 ] }
-multihash = { version = "0.18.1", default-features = false, features = [
-     "multihash-impl",
-] }
-libipld-core = { version = "0.13.1", features = ["serde-codec"] }
+multihash = { version = "0.19.1", default-features = false }
+multihash-codetable = { version = "0.1.4", default-features = false }
+multihash-derive = { version = "0.9.1", default-features = false }
+ipld-core = { version = "0.4.1", features = ["serde"] }
 integer-encoding = { version = "3.0.3", default-features = false }
 
 # helix-onchain
-fvm_actor_utils = "11.0.0"
-frc42_dispatch = "7.0.0"
-frc46_token = "11.0.0"
+#fvm_actor_utils = "11.0.0"
```

Comment:
It doesn't work. I believe it's because of how we build the actor bundle for wasm? It's some special custom sauce I really wish we could get rid of ("bundling" was a running not-so-funny joke on the FVM team).